src/faebryk/libs/util.py

```diff
@@ -1135,3 +1135,29 @@ def run_live(
         )
 
     return "\n".join(stdout), process
+
+
+class CouldntOSOpen(Exception):
+    """Raised when we can't open a file."""
+
+
+def os_open(path: os.PathLike):
```

Comment:
I'm not sure what the linux equivalent for this is, but perhaps also a viable replacement for the KiCAD pcbnew opener, if we'll keep it

Reply:
xdg-open